docs/site/Behind-the-scene.md

```diff
@@ -1,22 +1,75 @@
 ---
 lang: en
-title: 'Components'
-keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI
+title: 'Behind the scene'
```

Comment:
Behind the scenes?

Reply:
Yeah, this is inconsistent in our current doc version. I'll fix the sidebar item to match the page title, which is "Behind the scenes". In the future, I'd like us to find a more fitting name if possible.